src/NCronJob/Configuration/Builder/NCronJobOptionBuilder.cs

```diff
@@ -107,19 +107,11 @@ public NCronJobOptionBuilder AddExceptionHandler<TExceptionHandler>() where TExc
         return this;
     }
 
-    IRuntimeJobBuilder IRuntimeJobBuilder.AddJob<TJob>(Action<JobOptionBuilder>? options)
-    {
-        AddJob<TJob>(options);
-        return this;
-    }
+    void IRuntimeJobBuilder.AddJob<TJob>(Action<JobOptionBuilder>? options) => AddJob<TJob>(options);
```

Comment:
same question here.

And (I haven't checked), isn't there any already existing doco that showcase that non longer chaining capability?

Reply:
In the docs for that specific scenario there wasn't any chaining.